.github/actions/next-stats-action/src/prepare/repo-setup.js

```diff
@@ -84,7 +84,7 @@ module.exports = (actionInfo) => {
           pkgs = await fs.readdir(path.join(repoDir, 'packages'))
         } catch (err) {
           if (err.code === 'ENOENT') {
-            console.log('no packages to link')
+            require('console').log('no packages to link')
```

Comment:
What's the `require('console')` for in this case?

Reply:
In tests, you have that ugly formatting if you use `console.log`:
```
console.log
    exec: ls /tmp/next-repo-dc2a94e75cefe2bf100b504a57d9b198a47f61d47812d114bad947d45067c8f4/packages/next-swc/native
```
You need to use unpatched `require("console")` to get around it. It's even worse with objects because it uses pretty print, which takes a lot of screen space.